spec/controllers/events_controller_spec.rb

```diff
@@ -239,6 +239,31 @@
     end
   end
 
+  describe "POST #download_material" do
+    before :each do
+      filepath = Rails.root.join('spec/testfiles/actual.pdf')
+      @file = fixture_file_upload(filepath, 'application/pdf')
+      @event = Event.create! valid_attributes
+      post :upload_material, event_id: @event.to_param, session: valid_session, file_upload: @file
+    end
+
+    after :each do
+      filepath = File.join(@event.material_path, @file.original_filename)
+      File.delete(filepath) if File.exist?(filepath)
+    end
+
+    it "download a file from the event's material directory" do
+      post :download_material, event_id: @event.to_param, session: valid_session, file: @file.original_filename
+      response.header['Content-Type'].should eql 'application/pdf'
```

Comment:
This causes a deprecation warning whenever running tests. Use expect(...).to eq (...) instead.